Index: .idea/.idea.Week02_Test.dir/.idea/indexLayout.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.idea.Week02_Test.dir/.idea/indexLayout.xml b/.idea/.idea.Week02_Test.dir/.idea/indexLayout.xml
new file mode 100644
--- /dev/null	(date 1776843413980)
+++ b/.idea/.idea.Week02_Test.dir/.idea/indexLayout.xml	(date 1776843413980)
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="UserContentModel">
+    <attachedFolders />
+    <explicitIncludes />
+    <explicitExcludes />
+  </component>
+</project>
\ No newline at end of file
Index: .idea/.idea.Week02_Test.dir/.idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.idea.Week02_Test.dir/.idea/.gitignore b/.idea/.idea.Week02_Test.dir/.idea/.gitignore
new file mode 100644
--- /dev/null	(date 1776843419533)
+++ b/.idea/.idea.Week02_Test.dir/.idea/.gitignore	(date 1776843419533)
@@ -0,0 +1,15 @@
+# 디폴트 무시된 파일
+/shelf/
+/workspace.xml
+# Rider에서 무시된 파일
+/projectSettingsUpdater.xml
+/contentModel.xml
+/.idea.Week02_Test.iml
+/modules.xml
+# 에디터 기반 HTTP 클라이언트 요청
+/httpRequests/
+# 쿼리 파일을 포함한 무시된 디폴트 폴더
+/queries/
+# Datasource local storage ignored files
+/dataSources/
+/dataSources.local.xml
